src/components/IconCircle.tsx

```diff
@@ -30,8 +30,8 @@ export function IconCircle({
         a.align_center,
         a.rounded_full,
         {
-          width: 64,
-          height: 64,
+          width: size === 'lg' ? 52 : 64,
+          height: size === 'lg' ? 52 : 64,
```

Comment:
This will only affect this component, we should come back and create different size categories for this @estrattonbailey